src/plot/plaxis.cpp

```diff
@@ -694,17 +705,18 @@ void wxPLTimeAxis::RecalculateTicksAndLabel() {
         // Less than 2 days.  Need to label time.
 
         wxDateTime timeKeeper2(timeKeeper.GetTicks());
-        timeKeeper2.Add(wxTimeSpan::Minutes(60 * world_len));
+        timeKeeper2.Add(wxTimeSpan::Minutes(60 * world_len)); // TODO 289 - check scaling
         timeKeeper2.Subtract(wxTimeSpan::Minute()); //If it is 0:00 the next day, its really the same day.
         if (timeKeeper.IsDST() && !timeKeeper2.IsDST()) {
             timeKeeper.Add(wxTimeSpan::Hour());
             timeKeeper2.Add(wxTimeSpan::Hour());
         }
-
         if (timeKeeper.GetDay() == timeKeeper2.GetDay())
             m_timeLabel = timeKeeper.Format("%b %d");
+        else if (timeKeeper.GetMonth() == timeKeeper2.GetMonth())
+            m_timeLabel = timeKeeper.Format("%b %d") + "-" + timeKeeper2.Format("%d"); // TODO fix year 2 Feb 28-01 (always been the case)
```

Comment:
Same question on the TODO.

Reply:
Updated in https://github.com/NREL/wex/pull/179/commits/9f31b487cd0f51e1de6726f362e596e503eb157c